VictoriaAlbert/VictoriaAlbert/VAViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  VAViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Annie Tung on 11/10/16.
+//  Copyright © 2016 Annie Tung. All rights reserved.
+//
+
+import UIKit
+
+class VAViewController: UIViewController {
+
+    @IBOutlet weak var dateTextLabel: UILabel!
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var objectLabel: UILabel!
+    @IBOutlet weak var placeLabel: UILabel!
+    var victoria: VictoriaAlbert?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let validVC = victoria {
+            self.title = validVC.object
+        }
+       
+        objectLabel.text = victoria!.object
```

Comment:
I was glad to see you pick up some extra credit with these fields. You should have safely unwrapped the ```victoria``` property.